src/components/Button/Button.module.css

```diff
@@ -23,3 +23,7 @@
   background-color: var(--color-orange-button);
   color: var(--color-gray-text);
 }
+.green {
```

Comment:
Not sure whether it's desired to just keep yellow and orange buttons. I wanted to distinguish staking rewards with the green color branding.